Providers/nxOMSAuditdPlugin/MSFT_nxOMSAuditdPluginResource.h

```diff
@@ -19,7 +19,7 @@
 ** MSFT_nxOMSAuditdPluginResource [MSFT_nxOMSAuditdPluginResource]
 **
 ** Keys:
-**    Ensure
```

Comment:
You auto-generated this file, correct? If not, you should, because there's no real reason this file should be checked over by hand.